AsyncDisplayKit/ASTextNode.mm

```diff
@@ -401,7 +402,7 @@ - (CGSize)calculateSizeThatFits:(CGSize)constrainedSize
   size.width += (_textContainerInset.left + _textContainerInset.right);
   size.height += (_textContainerInset.top + _textContainerInset.bottom);
   
-  return size;
+  return CGSizeMake(fminf(size.width, constrainedSize.width), fminf(size.height, constrainedSize.height));
```

Comment:
Is this right? The constrained size is supposed to have the textContainerInset removed whereas the size shouldn't.